OrbitCore/CaptureData.h

```diff
@@ -103,6 +110,38 @@ class CaptureData {
   void UpdateFunctionStats(orbit_client_protos::FunctionInfo* func,
                            const orbit_client_protos::TimerInfo& timer_info);
 
+  [[nodiscard]] const CallstackData* GetCallstackData() const {
+    return callstack_data_.get();
+  };
+
+  void AddUniqueCallStack(CallStack&& call_stack) {
+    callstack_data_->AddUniqueCallStack(std::move(call_stack));
+  }
+
+  void AddCallstackEvent(
```

Comment:
As above, probably we want value args here.

Reply:
Done